eth/stagedsync/stage_mining_finish.go

```diff
@@ -46,6 +50,15 @@ func SpawnMiningFinishStage(s *StageState, tx kv.RwTx, cfg MiningFinishCfg, quit
 	block := types.NewBlock(current.Header, current.Txs, current.Uncles, current.Receipts)
```

Comment:
Uncles are prohibited after the Merge. I'm thinking that we probably should remove uncles altogether from our mining. It's unlikely that anybody will use Erigon for mining before the Merge (and even if they do, it's a small loss not to include uncles).